cmd/onboard_test.go

```diff
@@ -0,0 +1,191 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/operate-first/opfcli/constants"
+	"github.com/operate-first/opfcli/models"
+	"github.com/operate-first/opfcli/utils"
+	"github.com/stretchr/testify/require"
+)
+
+func (suite *commandTestSuite) TestOnboardWithQuota() {
+	assert := require.New(suite.T())
+
+	// Should fail with too few args
+	cmd := NewCmdOnboard(suite.api)
+	cmd.SetArgs([]string{})
+	err := cmd.Execute()
+	assert.EqualError(err, "accepts 1 arg(s), received 0")
+
+	// ---
+
+	// Should fail with too many args
+	cmd = NewCmdOnboard(suite.api)
+	cmd.SetArgs([]string{"arg1", "arg2", "arg3"})
+	err = cmd.Execute()
+	assert.EqualError(err, "accepts 1 arg(s), received 3")
+
+	// ---
+
+	// Should fail with unknown option
+	cmd = NewCmdOnboard(suite.api)
+	cmd.SetArgs([]string{"--failure", "arg1", "arg2", "arg3", "arg4", "arg5"})
+	err = cmd.Execute()
+	assert.EqualError(err, "unknown flag: --failure")
+
+	// ---
+
+	// Should fail with quota option used but no quota
+	cmd = NewCmdOnboard(suite.api)
+	cmd.SetArgs([]string{"base/testdata/Onboard/sampleOnboardConfigWithQuota.yaml"})
+	err = cmd.Execute()
+	assert.EqualError(err, "quota testquota does not exist")
```

Comment:
this always reports `testquota` does not exist, just reduce it to `quota does not exist`